docs/azure-pipelines-task.md

```diff
@@ -94,15 +94,31 @@ You can use a name other than `PR_Metrics_Access_Token` if you wish, but you
 will need to keep the name unified across the variable and task definitions. The
 name `PR_METRICS_ACCESS_TOKEN` cannot be altered.
 
-## Always Close Comment
+## Git History
+
+Depending on the nature of your repository history, it may be necessary to fetch
```

Comment:
Is source of the the issue leading to the particular repository history state known?
e.g. application of a particular merge strategy?
If so suggest adding a note about that here.

Reply:
Good idea. I've updated this with the details now.